config/grasp_debug_level.yaml

```diff
@@ -18,6 +18,7 @@ moveit_grasps:
     translation_x_score_weight: 0.0
     translation_y_score_weight: 0.0
     translation_z_score_weight: 4.0
+    grasp_approach_vector_in_parent_frame: [0, 0, 1]
```

Comment:
this does not go in grasp_debug_level, please move this now along with lines 13-20

Reply:
I think that renaming grasp_debug_level -> moveit_grasps_config addresses the issue of lines 13-20. I will talk to andy about the getting this transform out of the grasp_pose_to_eef_transform parameter